sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/parameterized_spark.py

```diff
@@ -44,17 +44,19 @@ class ParameterizedSpark(SparkJobEntryMixin):
 
     def __init__(
         self,
-        code: Union[str, os.PathLike] = ".",
-        entry: Union[Dict[str, str], SparkJobEntry, None] = None,
+        code: Optional[Union[str, os.PathLike]] = ".",
+        entry: Optional[Union[Dict[str, str], SparkJobEntry]] = None,
         py_files: Optional[List[str]] = None,
         jars: Optional[List[str]] = None,
         files: Optional[List[str]] = None,
         archives: Optional[List[str]] = None,
         conf: Optional[Dict[str, str]] = None,
         environment: Optional[Union[str, Environment]] = None,
         args: Optional[str] = None,
-        **kwargs,
+        **kwargs: Any,
     ) -> None:
+        self.args = None
```

Comment:
Why are we creating this `args` attribute? And why not set it equal to `args` instead of null?